packages/vscode.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,15 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+try {
+  $toolName = 'VSCode'
+  $category = 'Text Editors'
```

Comment:
We are not adding other Text Editors (like notepad++) to the Tools folder. I have no strong opinion if we should add all the Text Editors or not, but we should do this in a consistent way. @vm-packages opinions?

Related: https://github.com/mandiant/VM-Packages/issues/883

Reply:
agreed, no strong feelings either way - so let's do what is easiest to maintain...

Reply:
I'm all for throwing this into the `productivity tools` category of we go that route. 🙂